### DISCUSSION TITLE
How can i find the training details with pyannote-audio?

### DISCUSSION THREAD

**Discussion by xuanjihe** [pyannote/pyannote-audio#857] [Development]
Hi, I trained an overlap speech detection with my own data by pyannote-audio(version: 2.0) and get outstanding performance. Now I want to learn more about the details of the training, but I can't find the entry of the training script, can you tell me where the function entry corresponding to the bin of pyannote-audio is, so that I can better understand the training process?

**Comment by hbredin** [comment]
Thanks for the positive feedback! 

Would you mind sharing which command(s) you ran? Difficult to answer without this information.

**Reply by xuanjihe** [reply]
I use pyannote-audio to train an overlap speech detection model on my own data, and I run the command:
pyannote-audio ovl train --subset=train --to=1200 --parallel=20 --gpu --from 100 alimeet/ovd AMI.SpeakerDiarization.MixHeadset
